### PR TITLE
Tooltips on plan creation page

### PR DESCRIPTION
**Pull request by petsch0**
#412 #964

9d2909df-0681-48ff-a897-495056c78f5d (2x)

### CODE REVIEW

File: arbeitszeit_flask/templates/macros/draft_form.html [hunk 1]
```diff
@@ -39,18 +36,21 @@
 </div>
 <div class="block py-2"></div>
 <div class="box has-background-info-light">
-  <div class="has-text-right has-text-info-dark js-modal-trigger" data-target="id_unit_amount">
-    <span class="icon"><i class="fa-solid fa-circle-question"></i></span>
-  </div>
   <div class="field">
+    <div class="has-text-right has-text-info-dark">
+      {{ info_tooltip(id="id_unit", text=gettext("This value is used for calculation purposes only, i.e. you do not have to adhere to any rules here. For example, if you plan to produce beverages, you are free to choose if your smallest delivery unit is one can or one six-pack of cans."))}}
```
Comment (sloschert) [inline arbeitszeit_flask/templates/macros/draft_form.html:41]: It is actually more of an "informational purpose", I would say (you inform the potential consumers). It is, by the way, also relevant for cooperations, where the delivery units should match (we do not enforce this, though). 

What about this?

"This value is used for informational purposes only, i.e. you do not have to adhere to any rules here. For example, if you plan to produce beverages, your smallest delivery unit might be one can or one six-pack of cans."

File: arbeitszeit_flask/templates/macros/draft_form.html [hunk 2]
```diff
@@ -39,18 +36,21 @@
...
     <div class="control">
       {{ form.prd_unit(class_="input is-large") }}
-      <p class="help">{{ gettext("E.g. 1 package of 100 pieces, 1 kilogram, 1 language lesson of 1.5 hour, etc.")}}</p>
+      <p class="help">{{ gettext("E.g. 1 package, 1 kilogram, 20 bottles, 1 language lesson, etc.")}}</p>
     </div>
   </div>
   <div class="block"></div>
   <div class="field">
+    <div class="has-text-right has-text-info-dark">
+      {{ info_tooltip(id="id_amount", text=gettext("This value is assumed to be in the same unit as Smallest delivery unit. By dividing the sum of your production costs through this amount, the app calculates how much labour time is required to produce one piece of your product."))}}
```
Comment (sloschert) [inline arbeitszeit_flask/templates/macros/draft_form.html:52]: "Smallest" -> "smallest"
"divide BY"

What about changing the first sentence?

'How many units, as described in "smallest delivery unit", will you produce? By dividing the sum of your production costs by this amount, the app calculates how much labour time is required to produce one piece of your product.'

**Review by sloschert — CHANGES_REQUESTED** [review]
Great, thanks for the change. I propose some changes, please have a look if this makes sense for you. 

(Not in the scope of this Pull Request): In the last pull requests we added quite some additional (English) text. We should not wait too long with translating these texts into German. If you are interested in an intro into the translation pipeline let me know, we can have a call or meet for that purpose.

File: arbeitszeit_flask/templates/macros/draft_form.html (hunk 1 above)
Comment (petsch0) [reply arbeitszeit_flask/templates/macros/draft_form.html]: done. The info about cooperations is maybe sth for the "help" docu?

File: arbeitszeit_flask/templates/macros/draft_form.html (hunk 2 above)
Comment (petsch0) [reply arbeitszeit_flask/templates/macros/draft_form.html]: done, changed first sentence to "How many of your \"smallest delivery unit\" will you produce in total?"

File: arbeitszeit_flask/templates/macros/draft_form.html (hunk 1 above)
Comment (sloschert) [reply arbeitszeit_flask/templates/macros/draft_form.html]: Yes, the whole cooperation thing is still missing there.

**Comment by sloschert** [comment]
Thanks. If you want to, let us know your plan and how long it took.

**Review by sloschert — APPROVED** [review]

**Comment by sloschert** [comment]
2x consumption registered

**Pull request merged**